src/Exception/InvalidArgumentException.php

```diff
@@ -1,14 +0,0 @@
-<?php
-
-namespace Http\Client\Exception;
-
-use Http\Client\Exception;
-
-/**
- * Custom Invalid Argument exception
- *
- * @author Márk Sági-Kazár <mark.sagikazar@gmail.com>
- */
-final class InvalidArgumentException extends \InvalidArgumentException implements Exception
```

Comment:
removing this means a user can not catch this specific exception anymore. but as this is a programming error we might not need it.